cymetric/brightlite_root_metrics.py

```diff
@@ -0,0 +1,10 @@
+from cymetric.root_metrics import root_metric
+
+agentstate_brightlite_fuelfabfacilityinfo = root_metric(
```

Comment:
We could test them but I didn't since the idea was to move this out of cymetric in two stages (separate it out from the main files, then next figure out where to keep this type of file) I didn't write any tests.